From 2b5de029af27bd5c76babb770a687019549c652a Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Wed, 26 Feb 2020 08:01:42 +0200
Subject: [PATCH 2/3] Revert "add GoogleSans font normally"

This reverts commit 3e4b102991f826fdb9290c64de9d4b078b75bf40.

	modified:   target/product/handheld_system.mk
---
 target/product/handheld_system.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/target/product/handheld_system.mk b/target/product/handheld_system.mk
index bf5c7f9..5ebceea 100644
--- a/target/product/handheld_system.mk
+++ b/target/product/handheld_system.mk
@@ -30,7 +30,6 @@ $(call inherit-product-if-exists, external/google-fonts/arvo/fonts.mk)
 $(call inherit-product-if-exists, external/google-fonts/lato/fonts.mk)
 $(call inherit-product-if-exists, external/google-fonts/rubik/fonts.mk)
 $(call inherit-product-if-exists, external/google-fonts/zilla-slab/fonts.mk)
-$(call inherit-product-if-exists, external/googlesans-fonts/fonts.mk)
 $(call inherit-product-if-exists, external/noto-fonts/fonts.mk)
 $(call inherit-product-if-exists, external/roboto-fonts/fonts.mk)
 $(call inherit-product-if-exists, external/hyphenation-patterns/patterns.mk)
-- 
2.17.1

